spec/models/event_spec.rb

```diff
@@ -20,6 +20,31 @@
     expect(event).to be_valid
   end
 
+  it "checks if there are unclassified applications_letters" do
+    event = FactoryGirl.create(:event)
+    acceptedApplicationLetter = FactoryGirl.create(:application_letter, :event => event, :user => FactoryGirl.create(:user), :status => 1)
+    event.application_letters.push(acceptedApplicationLetter)
+    expect(event.applicationsClassified?).to eq(true)
+
+    rejectedApplicationLetter = FactoryGirl.create(:application_letter, :event => event, :user => FactoryGirl.create(:user), :status => 2)
```

Comment:
I think the name rejectedApplicationLetter is misleading, isnt it rather an application letter which is still pending (since you use status: 2)?